src/config/services/index.md

```diff
@@ -3,19 +3,33 @@
 Void uses the [runit(8)](https://man.voidlinux.org/runit.8) supervision suite to
 run system services and daemons.
 
+## Enable
```

Comment:
[2] Personally, I think this page does not need this type of thing in it's own section; it's in the context of the page already.

Reply:
I'm actually not too opposed to this, but more details per subsection would be nice if we did this. Like mentioning that the service will automatically be started.